packages/components/src/atoms/Loader/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './Loader'
+export type { LoaderProps } from './Loader'
```

Comment:
<img width="153" alt="Screen Shot 2023-01-18 at 20 08 01" src="https://user-images.githubusercontent.com/11325562/213314951-4ca0376f-5aca-4faa-8365-940ac6213411.png">

GH is complaining.

```suggestion
export type { LoaderProps } from './Loader'

```